File: Interview_Prep__App/native_java/src/main/java/com/dreams/androidquizapp/repository/reponseModels/AnswersResponse.java
package com.dreams.androidquizapp.repository.reponseModels;

import com.dreams.androidquizapp.repository.models.Answer;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Tc2r on 5/19/2017.
 * <p>
 * Description:
 */

public class AnswersResponse {
    @SerializedName("answers")
    private ArrayList<Answer> answersList;

    public ArrayList<Answer> getAnswersList() {
        return answersList;
    }

}

File: Interview_Prep_App/native_kotlin/retrofit/src/main/java/com/interviewprep/kotlinretrofit/repository/models/response/AnswersResponse.kt
package com.interviewprep.kotlinretrofit.repository.models.response

import com.google.gson.annotations.SerializedName
import com.interviewprep.kotlinretrofit.repository.models.Answer

data class AnswersResponse(
    @SerializedName("answers")
    var answers: ArrayList<Answer>
)
